docs/List.info.json

```diff
@@ -0,0 +1,282 @@
+{
```

Comment:
Cleanup files there please...

Reply:
If these are not supposed to be here then we should add them to gitignore.